spec.html

```diff
@@ -33759,7 +33767,7 @@ <h1>String.prototype.lastIndexOf ( _searchString_ [ , _position_ ] )</h1>
           1. Let _start_ be the result of clamping _pos_ between 0 and _len_.
           1. If _searchStr_ is the empty String, return 𝔽(_start_).
           1. Let _searchLen_ be the length of _searchStr_.
-          1. For each non-negative integer _i_ starting with _start_ such that _i_ &le; _len_ - _searchLen_, in descending order, do
+          1. For each non-negative integer _i_ such that _start_ &le; _i_ &le; _len_ - _searchLen_, in descending order, do
```

Comment:
This is wrong - because this is in _descending_ order, `_start_` is (despite appearances) an _upper_ bound; the lower bound is 0, given by the "non-negative" part.

I think instead where we set `_start_` to the result of clamping `_pos_` below `_len_`, we should instead clamp it below `_len_ - _searchLen_`. Then this can become

```suggestion
          1. For each integer _i_ such that 0 &le; _i_ &le; _start_, in descending order, do
```
which is more reasonable.

Reply:
Accepted your suggestion.